Assets/Scripts/PlayerControl.cs

```diff
@@ -242,14 +242,29 @@ IEnumerator Magnet()
 
     private void OnCollisionEnter(Collision other)
     {
-        if (other.gameObject.CompareTag("Enemy") || other.gameObject.CompareTag("Obstacle"))
+        if (other.gameObject.CompareTag("Enemy"))
         {
             if (isInvincible)
             {
                 Destroy(other.gameObject);
+                GameManager.inst.AddScore(1000); // 무적 상태에서 적 부딪하면 500점 추가
```

Comment:
```suggestion
                GameManager.inst.AddScore(1000); // 무적 상태에서 적 부딪하면 1000점 추가
```
이건 제가 후딱 수정 & 커밋 할게요